services/ext/api.go

```diff
@@ -1082,8 +1082,8 @@ func (api *PublicAPI) UnreadActivityCenterNotificationsCount() (uint64, error) {
 	return api.service.messenger.UnreadActivityCenterNotificationsCount()
 }
 
-func (api *PublicAPI) UnreadAndAcceptedActivityCenterNotificationsCount() (uint64, error) {
-	return api.service.messenger.UnreadAndAcceptedActivityCenterNotificationsCount()
+func (api *PublicAPI) UnreadAndAcceptedActivityCenterNotificationsCount(activityTypes []protocol.ActivityCenterType) (uint64, error) {
```

Comment:
@MishkaRogachev I think you recently changed `status-desktop` to consume this endpoint, correct? If that's true then you'll probably need to do some changes on the desktop client once you upgrade `status-go`.

Reply:
I've recently reverted it to `unreadActivityCenterNotificationsCount` for showing unread notifications counter on the bell icon because we are still implementing old designs for v1.0